deployer/deployer.yaml

```diff
@@ -67,13 +66,35 @@ items:
       - watch
       - delete
       - update
+  -
+    apiVersion: v1
+    kind: RoleBinding
+    metadata:
+      name: logging-deployer-edit-role
+    roleRef:
+      kind: ClusterRole
+      name: edit
+    subjects:
```

Comment:
so this lets us avoid needing to manually grant the deployer serviceaccount these roles?